src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -612,6 +638,29 @@ private async Task Orphan(IHost instance, CancellationToken cancellationToken =
             }
         }
 
+        private bool IsShutdownRequested()
+        {
+            return _applicationStopping == 1 || _applicationStopped == 1;
+        }
+
+        private bool NoOtherRestartIsInProgress()
```

Comment:
Could this be a property RestartRequested?

Reply:
I'm already using _restartRequested with value 0 or 1, another upper case property could get confusing?

---
In reply to: [369882227](https://github.com/Azure/azure-functions-host/pull/5437#discussion_r369882227) [](ancestors = 369882227)

Reply:
Just remove this method completely. You're already doing an inline Interlocked.Exchange in the RestartHostAsync method anyways.

Reply:
Changed it to property. Except I called it NoOtherRestartRequested, hope that conveys it's purposed..